backend/src/test/java/com/redhat/cloud/notifications/routers/internal/InternalResourceTest.java

```diff
@@ -0,0 +1,74 @@
+package com.redhat.cloud.notifications.routers.internal;
+
+import com.redhat.cloud.notifications.MockServerConfig;
+import com.redhat.cloud.notifications.TestHelpers;
+import com.redhat.cloud.notifications.TestLifecycleManager;
+import com.redhat.cloud.notifications.db.DbIsolatedTest;
+import io.quarkus.test.common.QuarkusTestResource;
+import io.quarkus.test.junit.QuarkusTest;
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+import org.junit.jupiter.api.Test;
+import java.time.LocalTime;
+import java.time.ZoneOffset;
+
+import static com.redhat.cloud.notifications.Constants.API_INTERNAL;
+import static com.redhat.cloud.notifications.TestHelpers.createTurnpikeIdentityHeader;
+import static io.restassured.RestAssured.given;
+import static io.restassured.http.ContentType.JSON;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@QuarkusTest
+@QuarkusTestResource(TestLifecycleManager.class)
+public class InternalResourceTest extends DbIsolatedTest {
```

Comment:
We already have a class named `InternalResourceTest` [here](https://github.com/RedHatInsights/notifications-backend/blob/master/backend/src/test/java/com/redhat/cloud/notifications/routers/InternalResourceTest.java).

Could you please move your tests from this PR to that class?

Reply:
I did a mix, I included the new tests in legacy test class, but I also keep new test class location to fit to InternalResource.java location.

Reply:
Perfect 👍